### DISCUSSION TITLE
Visualization/decimals: Different number of decimals for y-axis values than for data point values (mouse-over on graph)

### DISCUSSION THREAD

**Discussion by Backpacker1969** [grafana/grafana#55712] [Dashboards, Panels & Visualizations]
Hi,

It would be super nice to have the possibility to configure the number of decimals on the y-axis independently from the number of decimals of the data points (mouse-over on the chart).

**Reason:** It is convenient to be able to observe the full precision of values when examining the graphs (mouse-over), but having full precision (many decimals) on the y-axis values is unnecessary, takes up space and looks silly (e.g. 100000.0000).

It is currently possible to set the desired number of decimals in the Options Pane under Standard Option - > Unit -> Decimals. That kicks in in both of the above-mentioned places, which is what I find undesirable for many data sets.

Cheers,
Christian

P.S: In my specific case the visualization type I use is 'Time Series'.

**Comment by schwalbe13** [comment]
Yes please!

For illustration:

![image](https://github.com/grafana/grafana/assets/34183709/6daf2274-0b67-4701-a527-9cbf87d20ebc)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
